var/spack/repos/builtin/packages/cube/package.py

```diff
@@ -34,31 +34,37 @@ class Cube(AutotoolsPackage):
     """
 
     homepage = "http://www.scalasca.org/software/cube-4.x/download.html"
-    url = "http://apps.fz-juelich.de/scalasca/releases/cube/4.4/dist/CubeBundle-4.4.tar.gz"
+    url      = "http://apps.fz-juelich.de/scalasca/releases/cube/4.4/dist/cubegui-4.4.2.tar.gz"
 
-    version('4.4',   'f4f0544883bdd5bad9e11a4a692858bf')
+    version('4.4.2', '29b6479616a524f8325f5031a883963bf965fb92569de33271a020f08650ec7b')
+    version('4.4',   '0620bae3ac357d0486ce7f5f97e448eeb2494c9a31865b679380ee08c6750e70')
     version('4.3.5', 'e5dce986e3c6381ea3a5fcb66c553adc')
     version('4.3.4', '50f73060f55311cb12c5b3cb354d59fa')
     version('4.3.3', '07e109248ed8ffc7bdcce614264a2909')
-
     version('4.2.3', '8f95b9531f5a8f8134f279c2767c9b20')
 
-    variant('gui', default=False, description='Build CUBE GUI')
+    variant('gui', default=True, description='Build Cube GUI')
```

Comment:
can you even build the gui without the gui now? ;-)

Reply:
You can build older versions (< 4.4) without the GUI, since they use the bundled tarballs.